### DISCUSSION TITLE
Change WLAN configuration over WebIf

### DISCUSSION THREAD

**Discussion by FrodoVDR** [jomjol/AI-on-the-edge-device#2536] [Ideas]
It has happened to me several times now that I had to change the SSID and the password. Unfortunately, this is currently only possible by removing the SD card and changing the wlan.ini.
At least I have not found any other way. Uploading a wlan.ini with changed data and then rebooting unfortunately changes nothing.
Whereas I would see a change of the WLAN settings via the web frontend as a useful extension.

**Comment by caco3** [comment]
This is not possible to avoid by accident. You need to put the SD-card onto a PC to change it.

**Comment by teixeluis** [comment]
@FrodoVDR actually there is a way (which can be a bit of a life saver if you don't have physical access to the device and need to fix something on the wlan.ini file). Create a zip file with just the wlan.ini on it, and send an OTA update with that zip file. The update process will the overwrite that file on the MicroSD card and do nothing else.

**Reply by SirNoName2705** [reply]
Thanks so much, I will try that.
There is an open issue that would be resolved if this works and is more widely known.
For me, it is less about changing Wi-Fi password and more on setting IPs on static, like with the rest of my IOT network.
Unfortunately, I did not think of that before setting them up.

**Reply by BOFH90** [reply]
Sadly it didn´t worked on my device with V16. The html folder went missing after the zipped-wlan.ini-only-update. It worked with taking the latest update.zip and adding the wlan.ini to it.
